week6-rnn-tensorflow/01b_rnn_flask_js/static/sketch.js

```diff
@@ -35,30 +38,24 @@ function generate() {
   select('#length').html(lengthSlider.value())
   select('#temperature').html(tempSlider.value())
 
+  // Which model to use
+  modelToUse = document.querySelector('input[name="model"]:checked').value;
```

Comment:
In theory there's a `p5.Element` wrapper for a radio button, I wonder if it would work to use `select()` like above?